src/FidgetSpinner.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled, { css, keyframes } from 'styled-components'
+import img from './spinner.png';
+
+const rotate = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const animation = props =>
+  css`
+    ${rotate} ${props.speed}s linear infinite;
+  `
```

Comment:
It feels a little weird to see this with no closing semicolon (even though it's allowed...).
Since we have no style guide it's fine though!